README.md

```diff
@@ -40,3 +41,47 @@ In your eslint.json file enable this rule with the following:
     }
 }
 ```
+
+## Advanced configuration
+
+```js
+{
+    "plugins": ["no-unsanitized"],
+    "env": {
+        // $ruleName is either "method" or "property"
+        "no-unsanitized/$ruleName": ["errorlevel",
```

Comment:
Well, let's put the schema in a separate doc and have the README contain an example.